sdk/nodejs/ecs/ec2service.ts

```diff
@@ -130,8 +131,13 @@ export interface EC2ServiceArgs {
      * Whether to enable Amazon ECS Exec for the tasks within the service.
      */
     enableExecuteCommand?: pulumi.Input<boolean>;
+    /**
```

Comment:
I'm a little confused here, are automatically computing the AWSX schema based on the AWS schema it seems?

EC2ServiceArgs looks auto-generated in awsx/schema-types.ts 

This looks benign but is good to know.

Reply:
Yes, we ref types from the aws schema in the awsx schema. And we select the provider version for the schema based on the contents of `package.json`: https://github.com/pulumi/pulumi-awsx/blob/bef262a93d63ac8a5d728c0164999f5d03f581d8/schemagen/pkg/gen/schema.go#L38

Reply:
Hrm. We might be cloning the type contents instead of referencing types via schema references, but looks intentional